src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -3368,6 +3368,33 @@ End Namespace
             End Using
         End Function
 
+        <MemberData(NameOf(AllCompletionImplementations))>
+        <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestCompletionInsideMethodsWithDelegatesAsArguments(completionImplementation As CompletionImplementation) As Task
```

Comment:
We should also have a test with a delegate that isn't a generic delegate, i.e. explicitly have a delegate that hard-codes the type.

Reply:
I hope I added tests you mean. Please let me know if you see more tests to add.

---
In reply to: [276405342](https://github.com/dotnet/roslyn/pull/35067#discussion_r276405342) [](ancestors = 276405342)